From 76dda21a7dc612d7755c287348c52574ac35738f Mon Sep 17 00:00:00 2001
From: Todd Whiteman <todd.whiteman@joyent.com>
Date: Thu, 24 Aug 2017 14:19:46 -0700
Subject: [PATCH] DOCKER-1091 Cannot login into Artifactory private docker
 registry Reviewed by: Trent Mick <trentm@gmail.com> Approved by: Trent Mick
 <trentm@gmail.com>

---
 package.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package.json b/package.json
index c220bb5..96b024c 100644
--- a/package.json
+++ b/package.json
@@ -11,7 +11,7 @@
     "bunyan": "1.8.8",
     "cmdln": "3.2.1",
     "dashdash": "1.10.0",
-    "docker-registry-client": "3.2.6",
+    "docker-registry-client": "3.2.8",
     "effluent-logger": "git+https://github.com/joshwilsdon/effluent-logger.git#d662f161a07f94045ad2afb45442931511c40e51",
     "expiring-lru-cache": "2.1.0",
     "extsprintf": "1.2.0",
-- 
2.21.0

